app/controllers/api/v4/races/application_controller.rb

```diff
@@ -5,14 +5,36 @@ class Api::V4::Races::ApplicationController < Api::V4::ApplicationController
   before_action :set_raceable, only: %i[show update]
   before_action :check_permission, only: %i[update]
 
+  # Define all methods otherwise rubocop complains above
+  def index
+  end
+
+  def create
+  end
+
+  def show
+  end
+
+  def update
+  end
+
```

Comment:
Oops sorry for removing these, didn't realize they were for Rubocop.

Reply:
I didn't either until you removed it.  I just have a linter plugin for sublime that runs on every save so I try and stay on top of it in files I touch.